### DISCUSSION TITLE
After creation, how to use cocoapods in IOS project folder, or how to introduce an SDK package?

### DISCUSSION THREAD

**Discussion by wscart** [callstack/react-native-builder-bob#309] [Q&A]
# Question

These days can not find the relevant information, there is no way to understand thoroughly
The current problems are:

After creation, how to use cocoapods in IOS project folder, or how to introduce an SDK package?

![image](http://qinin.iut8.cn/1599102792187.jpg)

**Comment by krizzu** [comment]
I guess you want to run `pod init` in your iOS project? This will initialize `cocoapods` and create workspace
